packages/template/base/tmpl/.github/workflows/build.yml

```diff
@@ -0,0 +1,182 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Edge case: this should be adapted to the system value of `git config --global --get init.defaultBranch` with `main` as a fallback.

Reply:
Ooo, good catch, will amend this